src/rust/bin/inspect/transaction.rs

```diff
@@ -277,7 +285,7 @@ pub(crate) fn inspect(tx: Transaction, context: Option<Context>) {
 
                             if Some(txin.script_sig.0.len()) != script_len {
                                 eprintln!(
-                                    "  ⚠️  \"transparentcoins\" {} is P2PKH; txin {} scriptSig has length {} but data {}",
+                                    "    ⚠️  \"transparentcoins\" {} is P2PKH; txin {} scriptSig has length {} but data {}",
```

Comment:
These indentations were correct when I picked them; the warning triangle is meant to be positioned where a bullet point would be, not at the start of the paragraph indentation.

Reply:
The added indentation is to make them sub-bullets of the txid/index bullets.